README.md

```diff
@@ -1,4 +1,11 @@
 # UnPaSt
+[![Python Versions](https://img.shields.io/pypi/pyversions/unpast.svg)](https://pypi.org/project/unpast/)
```

Comment:
Hm, looks like tests are not run for all the python versions consistently. I'll fix that afterwards.

Reply:
What do you mean exactly? If what we have the highest version is 3.10, then it's not about testing - it's about the fact that we need to adjust package versions (like numpy or scikit-learn) to ensure comparability with higher versions. Or do you mean testing different python versions via github actions? 

Because this badge is from pypi, and pypi lists python versions based on toml file.

Reply:
Yeap, pypi reports several python versions, while only one is tested in run tests action (though others may be tested in other places).

Reply:
I think we can just fix yml test workflow with 

```
jobs:
  test:
    runs-on: ubuntu-latest
    strategy:
      matrix:
        python-version: [3.8, 3.9, 3.10]
```

I can add it in this branch

Reply:
> add it in this branch

done